Index: src/main/java/model/Board.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/model/Board.java b/src/main/java/model/Board.java
--- a/src/main/java/model/Board.java	
+++ b/src/main/java/model/Board.java	
@@ -161,5 +161,6 @@
 
         return true;
     }
+    
 
 }
\ No newline at end of file
